lib/mcb/commands/courses/sync_to_find.rb

```diff
@@ -23,11 +23,6 @@
     raise CriExitException.new(is_error: true)
   end
 
-  courses.each do |course|
-    ManageCoursesAPIService::Request.sync_course_with_search_and_compare(
-      user.email,
-      provider.provider_code,
-      course.course_code,
-    )
-  end
+  request = SearchAndCompareAPIService::Request.new
```

Comment:
Should this do a direct call or use the `BulkSyncCoursesToFindJob`

Reply:
Done